src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderImplTest {
+    private static final String CSV_INPUT
+            = "src/main/java/core/basesyntax/resources/reportToRead.csv";

```

Comment:
you should use separate test files. Don't use files from the base package for testing